data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,411 @@ mission "Ringworld Debris: Quarg"
+				accept
+	on enter "Patir"
+		dialog
+			`After a brief scan of the system, your ship's computers identify the asteroid that Dusk indicated you. Compared to the asteroids you are used to, this one seems orders of magnitude bigger.`
+
+event "asteroid on patir"
+	planet "Builder Settlement"
+		remove attributes "requires: ka'sei"
+
+mission "Ka'het: Patir Mystery 3"
+	landing
+	name "Return to <planet>"
+	description "Return to Dusk on <destination>."
+	source "Builder Settlement"
+	destination "Aventine"
+	cargo "short range scanner" 15
+	to offer
+		has "Ka'het: Patir Mystery 2: done"
+	on offer
+		conversation
+			`Your initial scans suggest that the asteroid is slowly rotating on itself. You set your ship on a polar orbit around it and activate the sensors that Dusk gave you, hopefully to find some answers to all the questions that Patir raised.`
+			`	You start your exploration from the dark side of the asteroid, and from the little you can see, the surface appears to be completely covered in regolith of little importance. But, soon after passing the terminator, you see an extremely reflective patch near the horizon, likely to be some sort of metal. As you get closer it becomes clear that this is the storage of iridium that Dusk was talking about: in front of you there is a huge reserve of the reflective metal you can't see the end of, extending for kilometers in every direction inside a gigantic hole the Ka'sei dug in the terrain. The metal appears to be magnetic, as when you start passing over it some of the instruments aboard your spaceship go haywire. Luckily, Dusk's scanners remain unaffected, allowing you to continue the exploration.`
+			`	In a few hours you are able to check every site you were told to scan, but most of the surface still remains an unknown.`
+			choice
+				`	(Continue your exploration of the asteroid.)`
+					goto explore
+				`	(Leave and return to Aventine.)`
+			`	As you are preparing to jump away, your eye is caught by an artificial structure near the edge of the iridium storage. The first thing you notice is an array of extremely high pylons put into two parallel lines; then, when you are able to get a better angle of view, you discover a large cupola towering all over the area. Although understanding its scale without a reference point is hard, you estimate it must be at least a kilometer in diameter.`
+				goto image
+			label explore
+			`	While you are setting up a new route to pass above as much terrain as possible, your eye is caught by an artificial structure near the edge of the iridium storage. The first thing you notice is an array of extremely high pylons put into two parallel lines; then, when you are able to get a better angle of view, you discover a large cupola towering all over the area. Although understanding its scale without a reference point is hard, you estimate it must be at least half a kilometer in diameter.`
+			label image
+			scene "scene/asteroid scene"
```

Comment:
Is there an asteroid scene? I don't see anything in this PR for it or in the game's scene folder.

Reply:
Whoops, I actually had the scene done before writing all this but it seems I forgot to paste it here when I PRed it
 I'll get to it this weekend when I'm back from holidays, if I haven't done it by saturday please remind me over the discord. In the meanwhile, old screenshot of it in game
![Screenshot_5579](https://user-images.githubusercontent.com/45330165/129770778-2eed1f33-b943-41b0-b84a-f593592ea09f.png)

Reply:
Will do - looks good!